op-e2e/actions/l2_verifier.go

```diff
@@ -31,10 +31,7 @@ import (
 type L2Verifier struct {
 	log log.Logger
 
-	eng interface {
-		engine.Engine
-		L2BlockRefByNumber(ctx context.Context, num uint64) (eth.L2BlockRef, error)
-	}
```

Comment:
I modified this so that I could use the existing eng from the L2Verifier for the NewL2Verifier creation. This is used in tests where a new node is "relaunched" while the execution engine stays the same. :)